stdlib/2/profile.pyi

```diff
@@ -0,0 +1,21 @@
+from typing import Any, Callable, Dict, Optional, TypeVar
+
+def run(statement: str, filename: Optional[str] = ..., sort: int = ...) -> None: ...
+def runctx(statement: str, globals: Dict[str, Any], locals: Dict[str, Any], filename: Optional[str] = ..., sort: int = ...) -> None: ...
+
+SelfT = TypeVar('SelfT', bound='Profile')
+T = TypeVar('T')
+
+class Profile:
+    def __init__(self, timer: Optional[Callable[[], int]] = ..., bias: Optional[int] = ...) -> None: ...
```

Comment:
The documentation implies that `timer` may return a `float`, though it doesn't explicitly say it. Can you verify this?